package/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+	"compilerOptions": {
+		"target": "ESNext",
+		"watch": true,
+		"lib": [
+			"ESNext",
+			"DOM"
+		],
+		"module": "ESNext",
+		"moduleResolution": "node",
+		"sourceMap": true,
+		"skipLibCheck": true,
+		"allowSyntheticDefaultImports": true
+	},
+	"exclude": [
+		"node_modules",
+		"./node_modules",
+		"./node_modules/*"
```

Comment:
Why 3 times?

Reply:
Honestly, I don't know. I just took this from the typescript website, I'll fix it

Reply:
I think just the first line is enough.